pkgs/development/node-packages/generate.sh

```diff
@@ -6,6 +6,6 @@ node2nix=$(nix-build ../../.. --no-out-link -A nodePackages.node2nix)
 
 cd ${DIR}
 rm -f ./node-env.nix
-for version in 10 12 13; do
+for version in 12 13; do
```

Comment:
Hm, maybe we could drop nodejs-13 and use nodejs-14 instead? (That is we're keeping LTS and latest stable).

```suggestion
for version in 12 14; do
```